### ISSUE TITLE
about reset

### ISSUE THREAD

**Issue by rockit-ba** [shepmaster/twox-hash#80]
about reset

```rust
let mut hash = XxHash64::with_seed(0);
hash.write(...);
// reset here
let mut hash = XxHash64::with_seed(0);
```

_Originally posted by @shepmaster in https://github.com/shepmaster/twox-hash/issues/72#issuecomment-879979172_

How much better is it to recreate an object than to reset the internal value of a hash？

**Comment by shepmaster** [comment]
I don't understand the question. Can you rephrase it? 

Do you believe there's a difference between recreating the object and "reset the internal value of a hash"?

**Comment by rockit-ba** [comment]
yes ，if in the Java ，there's a difference between recreating the object and "reset the internal value of a hash"?&nbsp;





------------------ Original ------------------
From: Jake Goulding ***@***.***&gt;
Date: Sat,Nov 6,2021 8:43 AM
To: shepmaster/twox-hash ***@***.***&gt;
Cc: 今天の吉祥君 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [shepmaster/twox-hash] about reset (Issue #80)





 
I don't understand the question. Can you rephrase it?
 
Do you believe there's a difference between recreating the object and "reset the internal value of a hash"?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
